tests/acceptance/features/webUIManageUsersGroups/deleteUsers.feature

```diff
@@ -22,4 +22,17 @@ Feature: delete users
 
   Scenario: use the webUI to cancel deletion of user
     When the administrator deletes user "user1" using the webUI and cancels the deletion using the webUI
-    Then user "user1" should exist
\ No newline at end of file
+    Then user "user1" should exist
+
+  Scenario Outline:  user names are not case-sensitive, deleteing users with different upper and lower case names
+    When the administrator creates a user with the name "<user_id1>" and the password "password" using the webUI
+    Then user "<user_id1>" should exist
+    And user "<user_id2>" should exist
+    When the administrator deletes user "<user_id1>" using the webUI and confirms the deletion using the webUI
```

Comment:
We could delete using a different case than used to create. That might break something ;)
```suggestion
    When the administrator deletes user "<user_id2>" using the webUI and confirms the deletion using the webUI
```

Reply:
@phil-davis we can't delete using different case than we used to create, because in webui users are listed in the same case in which we created them. For eg if we create a user ``` Phil```, this user is displayed as ```Phil``` and not as ```PHIL``` or ```phil```, so deleting in different case is not possible in webui.

Reply:
OK, I see now. In the webUI you select a row and delete. You do not actually type the user name in various mixed case.